content/intro-to-storybook/react/fr/using-addons.md

```diff
@@ -77,15 +115,15 @@ Nous pouvons maintenant reproduire et travailler sur ce cas limite avec facilit
 
 <video autoPlay muted playsInline loop>
   <source
-    src="/intro-to-storybook/task-stories-long-title.mp4"
+    src="/intro-to-storybook/task-stories-long-title-6-4.mp4"
     type="video/mp4"
   />
 </video>
 
 Si nous faisons un [test visuel](/intro-to-storybook/react/fr/test/), nous serons également informés si la solution elliptique se brise. Les cas limites obscurs sont susceptibles d'être oubliés sans la couverture de test!
```

Comment:
@xaviermarchal if you could rephrase this sentence and remove the reference to "obscurs" I'd appreciate it. The reasoning behind this is that that wording may have a different connotation. We could probably go with "Les cas extrêmes pourraient être ignorés sans couverture de test."

Reply:
I see, the initial translation did not catch my eye, as it sound french. I think it's better to translate "forgotten" by "oublié", because "ignoré" is more related to "ignored". 

=> I remove the term "obscure", does that sound ok for you?

Reply:
@xaviermarchal, that sounds reasonable to me. I agree with it.